src/programs.rkt

```diff
@@ -306,15 +300,28 @@
 (define (expand-parametric-reverse expr repr)
   (define ->bf (representation-repr->bf repr))
   (match expr
+    [(list 'if cond ift iff)
+     (define cond* (expand-parametric-reverse cond repr))
```

Comment:
Is this right? I assume `cond` should be unexpanded with the `boolean` representation?

Reply:
Comparison operators are also parameterized